research/flamby/fed_isic2019/moon/server.py

```diff
@@ -47,23 +53,35 @@ def main(config: Dict[str, Any], server_address: str) -> None:
         initial_parameters=get_initial_model_parameters(model),
     )
 
-    server = PersonalServer(client_manager, strategy)
+    server = FullExchangeServer(client_manager, model, strategy, checkpointer=checkpointer)
 
     fl.server.start_server(
         server=server,
         server_address=server_address,
         config=fl.server.ServerConfig(num_rounds=config["n_server_rounds"]),
     )
 
-    log(INFO, "Training Complete")
```

Comment:
No a big deal at all, but we lost the `log(INFO, "Training Complete")` here in the refactor.